src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequence.test.tsx

```diff
@@ -117,17 +157,22 @@ describe('<BlastInputSequence />', () => {
         value: 'AAACCCTTT'
       };
 
-      rerender(<BlastInputSequence {...commonProps} sequence={sequence} />);
+      const reRenderComponent = (
+        <Provider store={mockStore(mockState)}>
+          <TestContext.Provider value={testContext}>
+            <BlastInputSequence {...commonProps} sequence={sequence} />
+          </TestContext.Provider>
+        </Provider>
+      );
+      rerender(reRenderComponent);
```

Comment:
For these kind of rerenders what we have usually done in our tests is to render the component all over again like this:

```container = renderComponent({sequence}).container;```